src/generate_plot_data.py

```diff
@@ -8,8 +8,8 @@
 
 parser = argparse.ArgumentParser(description = 'Generates data for metric plots')
 parser.add_argument('-i', '--input-file', action = "store", help = 'path to the input results file')
-parser.add_argument('-o', '--output-dir', action = "store", help = 'path to put plot data', default = '../data/') #TODO: Can probably be smarter
-parser.add_argument('-n', '--num-boots', action = "store", help = 'select the number of bootstrap samples to take. Default 500', default = 500)
+parser.add_argument('-o', '--output-dir', action = "store", help = 'path to put plot data', default = './data/') #TODO: Can probably be smarter
```

Comment:
Is there a reason the base directory has changed?

Reply:
I had to change some of the inputs to only input integers, when I tried to change num-boots originally it wouldn't let me because it thought it was a string. I just changed the type to int.